troposphere/rds.py

```diff
@@ -18,7 +18,7 @@
                     'aurora-mysql', 'aurora-postgresql', 'mariadb')
 VALID_LICENSE_MODELS = ('license-included', 'bring-your-own-license',
                         'general-public-license', 'postgresql-license')
-
+VALID_ENGINEMODE = ('provisioned', 'serverless')
 
```

Comment:
I believe you need to add another blank line here to pass the commit tests. You can run "make test" locally to verify.